<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="ExceptionTest" tests="5" time="0.008" ignored="0" failures="0" timestamp="21 Jun 2018 07:13:03 GMT" skipped="0" hostname="asd1545po01976" errors="0">
  <testcase name="test_GameInProgressException" time="0.005" classname="ExceptionTest"/>
  <testcase name="test_NoUserWithSuchUUIDException" time="0.001" classname="ExceptionTest"/>
  <testcase name="test_PlayerExistsException" time="0.001" classname="ExceptionTest"/>
  <testcase name="test_maxPlayersReachedException" time="0.000" classname="ExceptionTest"/>
  <testcase name="test_InvalidTurnException" time="0.001" classname="ExceptionTest"/>
</testsuite> <!-- ExceptionTest -->
